modules/ossm-rn-known-issues.adoc

```diff
@@ -92,8 +92,65 @@ To check HTTPS or TCP traffic by using an ingress gateway, you must have an exis
 
 * https://issues.jboss.org/browse/MAISTRA-572[MAISTRA-572] Jaeger cannot be used with Kiali. In this release Jaeger is configured to use the OAuth proxy, but is also only configured to work through a browser and doesn't allow service access.  Kiali cannot properly communicate with the Jaeger endpoint and it considers Jaeger to be disabled.  See also https://issues.jboss.org/browse/TRACING-591[TRACING-591].
 
-* https://issues.jboss.org/browse/MAISTRA-622[MAISTRA-622]
-In Maistra 0.12.0/TP12, permissive mode does not work. The user has the option to use Plain text mode or Mutual TLS mode, but not permissive.
+* https://issues.jboss.org/browse/MAISTRA-622[MAISTRA-622] In Maistra 0.12.0/TP12, permissive mode does not work. The user has the option to use Plain text mode or Mutual TLS mode, but not permissive.
+
+* https://issues.jboss.org/browse/MAISTRA-684[MAISTRA-684] The Jaeger version in the `istio-operator` does not match the shipped version of Jaeger.
++
```

Comment:
"The Jaeger version in the `istio-operator` does not match the shipped version of Jaeger."
We should include information about the version that was in the operator and which version was shipped.

Reply:
We shipped version 1.13.1 and the default is 1.12.0

Reply:
I addressed this and will update the PR shortly.